lib/app_manifest.rb

```diff
@@ -36,9 +36,9 @@ def canonicalize_env(manifest)
       canonicalize_key(manifest, :env) do |env|
         Hash[
           env.map do |key, value|
-            if value.is_a? String
+            if !value.is_a?(Hash)
               value = {
-                value: value,
+                value: value.to_s,
```

Comment:
@gudmundur nyata validates (at the place you provided above), then coerces.
https://github.com/heroku/nyata/blob/master/lib/nyata/manifest/env.rb#L32

So a bool will fail creating the app-setup, as there will be errors. But we will also be able to parse the manifest file without causing errors.